ChangeLog

```diff
@@ -1,3 +1,8 @@
+2024-05-12  Adam Fox <adam.fox@keysight.com>
+
+	* Source/NSParagraphStyle.m (+defaultParagraphStyle): Restore
```

Comment:
This does not match the new implementation, but we just ignore that.

Reply:
Ah yeah, made sense in my head at the time but that's misleading, I'll change it.

Reply:
Okay @fredkiefer I updated the change description, please let me know if there's anything else, thank you!